lib/rubocop/cop/ezcater/rails_env.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Ezcater
+      class RailsEnv < Cop
```

Comment:
I was trying to avoid a very verbose name, but I'm open to better naming if this feels too generic.

Reply:
The name is fine with me.

Reply:
Also fine with me!